_includes/notebooks/2019-03-01-coupling-resolution.html

```diff
@@ -0,0 +1,123 @@
+<h3 id="Masks-and-Conservative-Regridding">Masks and Conservative Regridding<a class="anchor-link" href="#Masks-and-Conservative-Regridding">&#182;</a></h3><p>For conservative regridding to work, the masks of the two models must be consistent as well - there can't be areas where the ocean model thinks it's land and the atmosphere model thinks it's ocean.</p>
+<p>An example point where this can happen is the ice shelves around Antarctica. The ocean model's grid doesn't go all the way to the south pole, so it misses the southernmost part of the Ross ice shelf. The atmosphere grid does reach the pole however, and its default land mask assumes the ice shelf is ocean. There can be other areas where this causes conflicts as well - for instance the black sea is not included in the MOM default mask.</p>
+<p>To make sure that the grids are consistent you can do a conservative regridding of the mask from one grid to the other. The highest resoultion grid should be the source, in my case I used the ocean mask. I then conservatively regridded the <code>wet</code> field from MOM to the UM grid. It's important to not use masks for this interpolation, also since the MOM grid doesn't cover the entire globe I turned on the ESMF '--ignore_unmapped' flag. This gave me an ocean fraction field on the UM grid. The ocean fraction can then be straightforwardly converted to a land fraction and land mask field to be input to the UM.</p>
+<p>With a new land mask and land fraction the UM ancillaries needed to be modified</p>
+<h3 id="Regridding-Weights">Regridding Weights<a class="anchor-link" href="#Regridding-Weights">&#182;</a></h3><p>To calculate regridding weights I used <code>ESMF_RegridWeightGen</code>, a command-line tool included with ESMF. ESMF names the fields differently to SCRIP, so some modification of the output is required in order for OASIS to be able to read the files.</p>
+<p>Conservative regridding is used for scalar fields, patch regridding is used for vector field components.</p>
+
+</div>
+</div>
+</div>
+<div class="cell border-box-sizing text_cell rendered"><div class="prompt input_prompt">
+</div>
+<div class="inner_cell">
+<div class="text_cell_render border-box-sizing rendered_html">
+<h2 id="UM-Ancillary-interpolation">UM Ancillary interpolation<a class="anchor-link" href="#UM-Ancillary-interpolation">&#182;</a></h2><p>Since the UM is using a new resolution and mask the various ancillary datasets also need to be changed. For the most part files from the Met Office archive at the target resolution were used. For fields like vegetation fraction which are only defined on land points I 'de-masked' the anciallary field, by performing a nearest grid point interpolation from the masked source field to the same grid without a mask. It's important to use nearest grid point so that fields like the vegetation fractions correctly add up - some interpolation types also can result in negative values at some points which crashes the model.</p>
```

Comment:
typo "anciallary"
"For fields like vegetation fraction which are only defined on land points" as opposed to lat/lon? Either way, can you explain why you needed to do what you did?